converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Configurations/PatientConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.DicomCast.Core.Configurations;
+
+/// <summary>
+/// Configuration for Patient system Id
+/// </summary>
+public class PatientConfiguration
+{
+    /// <summary>
+    /// Patient System Id configured by the user
+    /// </summary>
+    public string PatientSystemId { get; set; }
+
+    /// <summary>
+    /// Issuer Id or Patient System Id used based on this boolean value
+    /// </summary>
+    public bool IsIssuerIdUsed { get; set; }
```

Comment:
This is not a great outcome haha. In the future, we cannot get into the habit where multiple PRs in flight and we cannot make changes to even simple parts of the design because it's already deployed. Otherwise, for all small changes we'll say, "well, 'x' is too small to warrant yet another deployment!" while bigger things will be met with, "that's too large of a change and we've already deployed! We'll get it in v2... maybe."

Reply:
Furthermore, are the health-paas changes and these changes so coupled? Are we passing values directly from the dicom spec into a configuration object?